src/lib/core/ripple/ripple-renderer.ts

```diff
@@ -182,14 +178,18 @@ export class RippleRenderer {
   /** Function being called whenever the trigger is being pressed. */
   private onMousedown(event: MouseEvent) {
     if (!this.rippleDisabled) {
-      this._isPointerDown = true;
-      this.fadeInRipple(event.clientX, event.clientY, this.rippleConfig);
+      this._isMousedown = true;
+      this.fadeInRipple(event.pageX, event.pageY, this.rippleConfig);
```

Comment:
Sorry, just took another look. This should use `clientX` and `clientY`.

Reply:
Good catch

Reply:
Updated it and fixed a lint error. Please have another look.